server/src/main/java/org/candlepin/audit/ArtemisMessageSourceReceiverFactory.java

```diff
@@ -147,10 +146,10 @@ else if (blacklist.size() > 0) {
         int maxJobs = config.getInt(ConfigProperties.ASYNC_JOBS_THREADS);
         log.info("Adding {} job receivers with filter: {}", maxJobs, this.amqpFilter);
 
-        for (int i = 0; i < maxJobs; i++) {
-            messageReceivers.add(
-                new JobMessageReceiver(this.amqpFilter, this.manager, sessionFactory, mapper));
-        }
+        // for (int i = 0; i < maxJobs; i++) {
```

Comment:
Can we just remove this code now (instead of commented out)?

Reply:
Sure